lib/commands/context.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Voicemail Administrator Context Commands.
+ *
+ * @module voicemail-admin
+ * @copyright 2015, Digium, Inc.
+ * @license Apache License, Version 2.0
+ * @author Jonathan R. Rose <jrose@digium.com>
+ */
+
+'use strict';
+
+var dal;
+var logger;
+var sprintf = require('sprintf').sprintf;
+
+function createCommands(dependencies) {
+  dal = dependencies.dal;
+  logger = dependencies.logger;
+
+  return {
+    showContexts: function(chosenCommand, entryComponents) {
```

Comment:
Just an idea to put in the future improvement bucket: the command implementations could have a comment above them that includes everything needed to generate the commands.json file using a grunt task. This way both the file needed for the CLI as well as the actual implementation would include the same documentation and adding/remove/editing commands could reflect changes in the commands file via a grunt task.